wile.py

```diff
@@ -43,7 +44,12 @@ def wile(obj, directory_url, staging, account_key_path, new_account_key_size, ve
 
     if staging:
         directory_url = 'https://acme-staging.api.letsencrypt.org/directory'
-    account_key = get_or_gen_key(account_key_path, new_account_key_size)
+
+    try:
+        account_key = get_or_gen_key(account_key_path, new_account_key_size)
+    except ValueError as e:
```

Comment:
oh, my bad, I thought this was about the Ctrl-C case, but it's actually about the case *after* the empty account key was already created. Now it makes sense! :D